### PR TITLE
wip - statsd - take 2

### PR DESCRIPTION
**Pull request by gorsuch**

### CODE REVIEW

**Comment by jmeichle** [comment]
as per #58 we should update the publishers section of the README

File: pkg/statsdpublisher/publisher.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	}
+
+	return New(source, host, prefix)
+}
+
+func (p *Publisher) Publish(m sensor.Measurement) error {
+	return Write(p.conn, p.Source, p.Prefix, m)
+}
+
+func Write(w io.Writer, source, prefix string, m sensor.Measurement) error {
```
Comment (jmeichle) [inline pkg/statsdpublisher/publisher.go]: with this, there appears no way to graph failures, or reason for failures either. we probably should graph pass/fail as well as each of the common reasons for failures.

Comment (blalor) [inline pkg/statsdpublisher/publisher.go]: What do you mean by "reasons"?  How do you translate an error string to a metric name?  Status codes are easy, but reasons for failure are not.

Comment (thomasjamesgoin) [inline pkg/statsdpublisher/publisher.go]: I would imagine response codes would be good enough.

**Pull request closed without merging**